odoo/cli/help.py

```diff
@@ -0,0 +1,28 @@
+from .command import Command, commands, load_addons_commands, load_internal_commands
+
+import odoo
+
+
+class Help(Command):
+    """ Display the list of available commands """
+
+    def run(self, args):
+        load_internal_commands()
+        load_addons_commands()
+
+        print(  # noqa: T201
```

Comment:
```suggestion
        print(
            f"usage: {prog_name} [--addons-path=PATH,...] {{{', '.join(sorted(commands))}}}\n"
            "\n"
```
```

Reply:
It's gonna be long and getting longer.

Alternative:
![immagine](https://github.com/user-attachments/assets/795313f6-60df-4aae-b158-2ffe467ee536)

Reply:
I'm fine with the alternative :)